pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -1283,7 +1283,16 @@ func (fs *eosfs) Delete(ctx context.Context, ref *provider.Reference) error {
 		return err
 	}
 
-	return fs.c.Remove(ctx, auth, fn)
+	return fs.c.Remove(ctx, auth, fn, false)
+}
+
+func getRootAuth() eosclient.Authorization {
```

Comment:
We already have a method for this

https://github.com/cs3org/reva/blob/92a58284fa90ff52d712aa46cf935e70c9aa8115/pkg/storage/utils/eosfs/eosfs.go#L1951